modules/ixBidAdapter.js

```diff
@@ -612,13 +612,15 @@ function buildRequest(validBidRequests, bidderRequest, impressions, version) {
     r.ext.ixdiag.err = cachedErrors;
   }
 
+  // populate request source.tid
+  r.source = {
+    tid: validBidRequests[0].transactionId,
```

Comment:
@robhazan what's your guys goal here? You already have this field on imp.ext.tid